Makefile

```diff
@@ -18,6 +18,12 @@ metalint_exclude    := .excludemetalint
 m3ctl_package       := github.com/m3db/m3ctl
 gopath_prefix       := $(GOPATH)/src
 vendor_prefix       := vendor
+mocks_output_dir 		:= generated/mocks/mocks
```

Comment:
These should be aligned with the rest of the assignment.

Reply:
Hmm this is super strange. It does not look like this locally. Ill figure out why this looks that way.

Reply:
There was a sneaky tab. Ill update that.